transifex/push.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+"""
+This script can be used to automatically extract translation strings,
+commit, push, and merge them to their respective repos, and then push them to transifex.
+
+To use, export an environment variable `GITHUB_ACCESS_TOKEN`. The token requires
+GitHub's "repo" scope.
+
+Run the script from the root of this repo.
+
+    python3 transifex/push.py git@github.com:edx/course-discovery.git
```

Comment:
This script is run using python3 on jenkins. Is it expected to be backwards compatible with python2 (I haven't tested that)? By using `python` here, are we communicating that the script should work with both python versions?